dds_web/commands.py

```diff
@@ -205,6 +205,39 @@ def update_unit(unit_id, sto4_endpoint, sto4_name, sto4_access, sto4_secret):
     gc.collect()
 
 
+@click.command("update-unit-quota")
+@click.option("--unit-id", "-u", type=str, required=True)
+@click.option("--quota", "-q", type=int, required=True)
+@flask.cli.with_appcontext
+def update_unit_quota(unit_id, quota):
+    """Update unit quota. The quota is in bytes."""
+    # Imports
+    import rich.prompt
+    from dds_web import db
+    from dds_web.database import models
+
+    # Get unit
+    unit: models.Unit = models.Unit.query.filter_by(public_id=unit_id).one_or_none()
+    if not unit:
+        flask.current_app.logger.error(f"There is no unit with the public ID '{unit_id}'.")
+        sys.exit(1)
+
+    # ask the user for confirmation
+    do_update = rich.prompt.Confirm.ask(
+        f"You are about to update the quota for unit '{unit_id}' to {quota}. Are you sure? \n"
+        f"Remember that the quota is in bytes. You have select about {round(quota / 1024 ** 3,2)} GB."
```

Comment:
Good idea on adding the confirmation. Just a little change though in my opinion
```suggestion
        f"You are about to update the quota for unit '{unit_id}' to {quota} bytes (approx {round(quota / 1024 ** 3,2)} GB). Are you sure? \n"
```

Reply:
Also, I like the use of the gibibyte calc, but in this case I think it'll be better to use gigabytes (1000 instead of 1024)
```suggestion 
        f"You are about to update the quota for unit '{unit_id}' to {quota} bytes (approx {round(quota / 1000 ** 3,2)} GB). Are you sure? \n"
```